nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateCsv.java

```diff
@@ -171,7 +174,16 @@ public class ValidateCsv extends AbstractProcessor {
             .description("Strategy to apply when routing input files to output relationships.")
             .required(true)
             .defaultValue(VALIDATE_WHOLE_FLOWFILE.getValue())
-            .allowableValues(VALIDATE_LINES_INDIVIDUALLY, VALIDATE_WHOLE_FLOWFILE)
+            .allowableValues(VALIDATE_LINES_INDIVIDUALLY, VALIDATE_WHOLE_FLOWFILE, VALIDATE_ATTRIBUTE_AS_CSV)
```

Comment:
Remove VALIDATE_ATTRIBUTE_AS_CSV from VALIDATION_STRATEGY and use the CSV_SOURCE_ATTRIBUTE property described below to trigger validating an attribute. Using CSV_SOURCE_ATTRIBUTE removes the need for VALIDATE_ATTRIBUTE_AS_CSV.  VALIDATE_ATTRIBUTE_AS_CSV does not seem to logically belong with the other two existing options in VALIDATION_STRATEGY.
Sidenote: Can the Attribute to Validate as CSV contain more than 1 line of csv data?

Reply:
I've made this change. Also, yes, the attribute can contain as many lines as the attribute value can hold.